core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -69,7 +69,7 @@ class ReligionManager {
     fun canFoundPantheon(): Boolean {
         if (!civInfo.gameInfo.hasReligionEnabled()) return false
         if (religionState != ReligionState.None) return false
-        if (!civInfo.isMajorCiv()) return false
+        if (civInfo.isCityState()) return false
```

Comment:
Doesn't that mean a Barbarian Religion? Do they at least get a large Bonus?

Reply:
They shouldn't generate faith so it should be fine in the base game, but I'll implement a check for it nontheless, in some weird mods where barbarians might earn faith by attacking this could cause problems

Reply:
Yeh - ***I*** could only be a Barbarian if I had _tons_ of faith - in _myself_ :smile_cat: